k8s.gcr.io/images/k8s-staging-etcd/images.yaml

```diff
@@ -1 +1,22 @@
-# No images yet
+- name: etcd
+  dmap:
+    "sha256:80d539b791923b37fb501166729307c987a237bf9a72cef3dbfca24744b5cb76": ["3.4.7-0", "3.4.7"]
+- name: etcd-amd64
```

Comment:
etcd-amd64, etcd-arm, etcd-arm64, etcd-ppc64le, etcd-s390x are all listed in the manifest for the `etcd` image:
https://console.cloud.google.com/gcr/images/k8s-staging-etcd/GLOBAL/etcd@sha256:80d539b791923b37fb501166729307c987a237bf9a72cef3dbfca24744b5cb76/details?tab=info

So we don't need to individually list them here in this yaml file.

Reply:
Thanks! I have a question on backward compatibility. If we only list `etcd` in this yaml, and user later ask for ` k8s.gcr.io /etcd-amd64:3.4.7`, will it work?

Reply:
@jingyih yep. that will work

Reply:
Thanks! PR updated.